ECAT7 support broken (compilation errors)

```
./src/STIR/src/IO/stir_ecat7.cxx:1383:45: error: there are no arguments to find_segment_sequence_in_stream that depend on a template parameter
```
bug introduced in #352 when renaming the function

Also
```
In file included from /slms/inm/research/tools/src/STIR/src/listmode_buildblock/CListModeDataECAT.cxx:28:0:
/slms/inm/research/tools/src/STIR/src/include/stir/listmode/CListRecordECAT966.h:169:1: error: expected class-name before  {  token
 {
```
bug introduced in https://github.com/UCL/STIR/pull/445.

--- a/src/IO/stir_ecat7.cxx
+++ b/src/IO/stir_ecat7.cxx
@@ -1380,7 +1380,7 @@ make_pdfs_from_matrix_aux(SUBHEADERPTR sub_header_ptr,
   pdi_ptr->set_num_axial_poss_per_segment(num_axial_poss_per_seg);
     
   std::vector<int> segment_sequence_in_stream =
-    find_segment_sequence_in_stream(*pdi_ptr);
+    find_segment_sequence(*pdi_ptr);
   
   Matval matval;
   mat_numdoc(matrix->matnum, &matval);

--- a/src/include/stir/listmode/CListRecordECAT962.h
+++ b/src/include/stir/listmode/CListRecordECAT962.h
@@ -29,6 +29,8 @@
 #define __stir_listmode_CListRecordECAT962_H__
 
 #include "stir/listmode/CListRecord.h"
+#include "stir/listmode/ListGatingInput.h"
+#include "stir/listmode/ListTime.h"
 #include "stir/listmode/CListEventCylindricalScannerWithViewTangRingRingEncoding.h"
 #include "stir/ProjDataInfoCylindrical.h"
 #include "stir/ProjDataInfoCylindricalNoArcCorr.h"
@@ -184,7 +186,7 @@ private:
 //! A class for a general element of a listmode file
 /*! \ingroup listmode
    For the 962 it's either a coincidence event, or a timing flag.*/
-class CListRecordECAT962 : public CListRecordWithGatingInput, public ListTime, public CListGatingInput,
+class CListRecordECAT962 : public CListRecordWithGatingInput, public ListTime, public ListGatingInput,
     public  CListEventCylindricalScannerWithViewTangRingRingEncoding<CListRecordECAT962>
 {
  public:
@@ -210,9 +212,9 @@ class CListRecordECAT962 : public CListRecordWithGatingInput, public ListTime, p
     { return *this; }
   virtual const ListTime&   time() const
     { return *this; }
-  virtual CListGatingInput&  gating_input()
+  virtual ListGatingInput&  gating_input()
     { return *this; }
-  virtual const CListGatingInput&  gating_input() const
+  virtual const ListGatingInput&  gating_input() const
   { return *this; }
 
   bool operator==(const CListRecord& e2) const

--- a/src/include/stir/listmode/CListRecordECAT966.h
+++ b/src/include/stir/listmode/CListRecordECAT966.h
@@ -29,6 +29,8 @@
 #define __stir_listmode_CListRecordECAT966_H__
 
 #include "stir/listmode/CListRecord.h"
+#include "stir/listmode/ListGatingInput.h"
+#include "stir/listmode/ListTime.h"
 #include "stir/listmode/CListEventCylindricalScannerWithViewTangRingRingEncoding.h"
 #include "stir/ProjDataInfoCylindrical.h"
 #include "stir/IO/stir_ecat_common.h" // for namespace macros
@@ -165,7 +167,7 @@ class CListTimeDataECAT966
 //! A class for storing and using a timing 'event' from a listmode file from the ECAT 966 scanner
 /*! \ingroup listmode
  */
-class CListTimeECAT966 : public ListTime, public CListGatingInput
+class CListTimeECAT966 : public ListTime, public ListGatingInput
 {
  public:
   Succeeded init_from_data_ptr(const void * const ptr)

